src/Api.php

```diff
@@ -958,11 +961,17 @@ public function setWebhook(array $params)
      *
      * @return Update
      */
-    public function getWebhookUpdates()
+    public function getWebhookUpdates($emitUpdateWasReceivedEvent = true)
```

Comment:
@irazasyed We need to have it, BUT the purpose of this param is the following:
Imagine if getWebhookUpdates is called twice... then events are emitted twice. What if developer does want it to happen just once? That's the purpose.